lib/symlink.go

```diff
@@ -32,45 +35,47 @@ func CreateSymlink(cwd string, dir string) {
 	} else {
 		err := os.Symlink(cwd, dir)
 		if err != nil {
-			logger.Fatalf(`
+			logger.Tracef(`
 		Unable to create new symlink.
 		Maybe symlink already exist. Try removing existing symlink manually.
 		Try running "unlink %s" to remove existing symlink.
 		If error persist, you may not have the permission to create a symlink at %s.
 		Error: %s
 		`, dir, dir, err)
-			os.Exit(1)
+			return errors.New(fmt.Sprintf("Unable to create new symlink %s : %s", dir, err))
 		}
 	}
+	return nil
 }
 
 // RemoveSymlink : remove symlink
-func RemoveSymlink(symlinkPath string) {
+func RemoveSymlink(symlinkPath string) error {
 
 	_, err := os.Lstat(symlinkPath)
 	if err != nil {
-		logger.Fatalf(`
+		logger.Tracef(`
```

Comment:
I think Debug has been preferred for these types of messages. Unless there's a specific reason that Trace is better for this type of message?